pkg/commands/cmdlib/bundleio.go

```diff
@@ -27,10 +27,11 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/files"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/inline"
+	"k8s.io/apimachinery/pkg/runtime"
 )
 
-// ReadBundle reads component data file contents from a file or stdin.
-func ReadBundle(ctx context.Context, rw files.FileReaderWriter, g *GlobalOptions) (*bundle.Bundle, error) {
+// readBytes reads data file contents from a file or stdin.
+func readBytes(ctx context.Context, rw files.FileReaderWriter, g *GlobalOptions) ([]byte, error) {
```

Comment:
Oh man, I need to create tests for this file so badly. On me, though. https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/issues/88